cmake/FindHunspell.cmake

```diff
@@ -0,0 +1,11 @@
+# - Try to find tree-sitter
```

Comment:
```suggestion
# - Try to find Hunspell
```

Reply:
and similarly for the other copypasta below ;)